WordPress/Classes/Networking/BlogServiceRemoteREST.m

```diff
@@ -125,12 +125,14 @@ - (void)updateBlogSettings:(RemoteBlogSettings *)remoteBlogSettings
                    failure:(void (^)(NSError *error))failure;
 {
     NSParameterAssert([blogID isKindOfClass:[NSNumber class]]);
-
     NSDictionary *parameters = @{ @"blogname" : remoteBlogSettings.name,
                                   @"blogdescription" : remoteBlogSettings.desc,
                                   @"default_category" : remoteBlogSettings.defaultCategory,
                                   @"default_post_format" : remoteBlogSettings.defaultPostFormat,
-                                  @"blog_public" : remoteBlogSettings.privacy
+                                  @"blog_public" : remoteBlogSettings.privacy,
+                                  @"jetpack_relatedposts_enabled" : remoteBlogSettings.relatedPostsEnabled,
```

Comment:
Good catch but I prefer to check them for nil here, or else we may end up changing the settings the user had set on the server.